framework/include/base/MooseApp.h

```diff
@@ -66,6 +68,14 @@ class MooseApp : public ConsoleStreamInterface,
                  public libMesh::ParallelObject
 {
 public:
+  struct FixedPointConfig
```

Comment:
This is just an artifact of needing to communicate multiapp input file config to the fixed point solve.  It may go away at some point.  But it is here for now and is stored and accessed via mooseapp - so may as well be defined there.